.changes/v2.15.0/440-improvements.md

```diff
@@ -0,0 +1,6 @@
+* NSX-T Edge Gateway now supports VDC Groups by switching from `OrgVdc` to `OwnerRef` field.
+  Additional methods `NsxtEdgeGateway.MoveToVdc()`, `Org.GetNsxtEdgeGatewayByNameAndOwnerId()`,
```

Comment:
`MoveToVdc` name is changed :) here update name also needed

Reply:
True.. Fixed.